service-sdk/16.0.0/create-outbound-integration-service-openapi/example-integration-openapi-service/pom.xml

```diff
@@ -10,19 +10,18 @@
     <parent>
         <groupId>com.backbase.buildingblocks</groupId>
         <artifactId>service-sdk-starter-core</artifactId>
-        <version>15.0.1</version>
+        <version>16.0.0</version>
         <relativePath />
     </parent>
 
     <groupId>com.backbase.example</groupId>
     <artifactId>example-integration-openapi-service</artifactId>
     <version>1.0.0-SNAPSHOT</version>
-    <packaging>war</packaging>
     <name>Backbase :: example-integration-openapi-service</name>
 
     <properties>
         <java.version>17</java.version>
-        <boat-maven-plugin.version>0.16.2</boat-maven-plugin.version>
+        <boat-maven-plugin.version>0.17.0</boat-maven-plugin.version>
```

Comment:
shall bump this to `0.17.5`? I know it's never going to be the most up to date but might as well grab the latest at this moment

Reply:
agree! updated!